ios/Podfile.lock

```diff
@@ -282,7 +282,7 @@ PODS:
     - Onfido (~> 28.3.0)
     - React
   - OpenSSL-Universal (1.1.1100)
-  - Plaid (4.7.0)
+  - Plaid (4.7.1)
```

Comment:
what's this for?

Reply:
Idk why it appeared here but i just run `pod install` so maybe someone else didn't do it even though he should have.

Reply:
I also run locally but still 4.7.0

Reply:
Did you remove previous Podfile.lock? I do it to be sure that it is always up to date.

Reply:
4.7.0 is the correct version
https://github.com/plaid/react-native-plaid-link-sdk/releases/tag/v10.8.0

Reply:
> Did you remove previous Podfile.lock? I do it to be sure that it is always up to date.

No. We should not remove previous lock file before pod install. Can you undo this change?

Reply:
Yeah, but it has `~>`: https://github.com/plaid/react-native-plaid-link-sdk/blob/23c5b8c2dcbdfe9532e70e4f92f1ef8b53da8b26/react-native-plaid-link-sdk.podspec#L19 which means it includes `4.7.1` is it was released. Or am I missing something?

Reply:
I hope there will be no regression in that minor version update but let's consider out of scope for now.
We don't remove lock files when install npm, pod. So 4.7.0 should be correct